wkcuber/api/TiffData/TiffMag.py

```diff
@@ -414,11 +414,11 @@ def open(cls, file_name):
         return cls(file_name)
 
     def read(self) -> np.array:
-        return io.imread(self.file_name)
+        return io.imread(self.file_name).transpose()
 
     def write(self, pixels):
         os.makedirs(os.path.dirname(self.file_name), exist_ok=True)
-        io.imsave(self.file_name, pixels, check_contrast=False)
+        io.imsave(self.file_name, pixels.transpose(), check_contrast=False)
```

Comment:
cc @rschwanhold: I'm confident that this fix is correct, but wanted to let you know, anyway (and feel free to doublecheck!). The bug is especially tricky to find, since it's intransparent when only using the API (since input and output are transposed, thus, effectively cancelling each other out). One has to inspect the tiff files by hand to see whether something is flipped or not. Of course, that's even harder to do, when the tests generate procedural images ^^